### ISSUE TITLE
Error message on atag.update(force=True)

### ISSUE THREAD

**Issue by gvugt** [EtxeanNet/atagone-mqtt-bridge#3]
Somehow for me it was not working, I got an error message on atag.update(force=True) ... 'force' was unknown ...
I removed force=True in atag_interaction.py ... now it is working!

I don't know whtat 'force' is doing ... ? Could I have solved it differently?

I'm a novice in programming, linux and github, so it could very well be that I did something wrong in installing the script... Still I wanted to let you know that I had this issue.

**Comment by ArdKuijpers** [comment]
Hi @gvugt, sorry, I totally forgot you added this issue. Now, another user found the same problem. The problem comes from a change in the pyatag library that I use, but which was changed in a not-backward compatible manner. I've looked into the new code and the force parameter has disappeared. It looks safe to remove it from the code.

I will change this and test it locally. If it works I'll close the issue.

You did nothing wrong. Thanks!
